### PR TITLE
Fix ChoiceTile content padding

### PR DESCRIPTION
**Pull request by PavelHolec**
image

image

### CODE REVIEW

**Comment by sjavora** [comment]
Is the badge shift intended/wanted?

Screenshot 2023-01-11 at 16 17 29

**Comment by PavelHolec** [comment]
> Is the badge shift intended/wanted?
> 
> Screenshot 2023-01-11 at 16 17 29

Should be fixed now. Updated the API to be more consistent with ListChoice, removing the (non-Orbit) badge and replaced by any custom header content.

File: Sources/Orbit/Components/ChoiceTile.swift [hunk 1]
```diff
@@ -186,12 +199,23 @@ public struct ChoiceTile<Content: View>: View {
         }
     }
     
-    var messagePadding: CGFloat {
-        indicatorSize > 0
-            ? (indicatorSize + padding + .xSmall)
+    var indicatorTrailingPadding: CGFloat {
+//        indicatorSize > 0 && alignment == .default && headerContent is EmptyView
```
Comment (sjavora) [inline Sources/Orbit/Components/ChoiceTile.swift:203]: Commented code.

**Comment by sjavora** [comment]
Also conflicts.

File: Sources/Orbit/Components/ChoiceTile.swift (hunk 1 above)
Comment (PavelHolec) [reply Sources/Orbit/Components/ChoiceTile.swift]: Thx, updated

**Review by sjavora — APPROVED** [review]

**Pull request merged**